proto/executor.proto

```diff
@@ -163,6 +187,7 @@ message Selection {
 	repeated Expr conditions = 1;
 	repeated RpnExpr rpn_conditions = 2;
 	optional Executor child = 3;
+	optional bool has_rf_conditions = 4 [(gogoproto.nullable) = false];
```

Comment:
Will `rf_condition` be put in `Selection`?

Reply:
Remove it.